core/src/com/fourinachamber/fortyfive/game/enemy/Enemy.kt

```diff
@@ -315,15 +319,28 @@ class EnemyActor(
         Label.LabelStyle(enemy.detailFont, enemy.detailFontColor)
     )
 
+    private val fireParticles: ParticleEffect by lazy {
+        ResourceManager.get(screen, "fire_particle") // TODO: fix
+    }
+
     init {
         healthLabel.setFontScale(enemy.detailFontScale)
         coverText.setFontScale(enemy.detailFontScale)
+        attackLabel.setFontScale(enemy.detailFontScale)
+
         image.setScale(enemy.scaleX, enemy.scaleY)
         image.reportDimensionsWithScaling = true
         image.ignoreScalingWhenDrawing = true
         coverIcon.setScale(enemy.coverIconScale)
+        attackIcon.setScale(enemy.coverIconScale) // TODO: fix
```

Comment:
i think its great that for a "rework" many "todo: fix" came (3 in this file at least). I don't have a problem with it, just find it interesting

Reply:
I needed to get the pull request out before i am on vacation lol